courses/templates/courses/introduction-to-data-mining.html

```diff
@@ -0,0 +1,148 @@
+<meta property="og:image:width" content="750"/>
+<meta property="og:image:height" content="303"/>
+<meta property="og:title" content="Workshop - Introduction to Data mining"/>
+<meta property="og:description"
+      content=""/>
+<meta property="article:publisher" content="https://www.facebook.com/orangedm/"/>
+<meta property="article:author" content="https://www.facebook.com/orangedm/"/>
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+{% endblock opengraph-social-image %}
+
+{% block text %}
+<img style="width:100%" src="{% static 'courses/dm-course-baylor2.jpg' %}" alt="hero shot"/>
+<h1>Introduction to Data Mining</h1>
+<p>Data science is an interdisciplinary field that deals with scientific methods, processes,
+    and systems to extract knowledge or insights from data. The course will introduce machine learning and
+    data visualization techniques to uncover hidden patterns in data. </p>
+
+<h2>Why should I attend?</h2>
+<ul style="margin-bottom: 2em;">
+    <li>You will learn essential approaches in data science that include data exploration, visualization,
+        unsupervised machine learning, classification and regression.
+    </li>
+    <li>After completing the course, you will be able to analyze your own data and use them to develop predictive
+        models.
+    </li>
+    <li>The course will be hands-on and will introduce data science concepts and methods through the analysis of
+        real-life data from biomedicine.
+    </li>
+</ul>
+
+<h2>What will I learn?</h2>
+<ul style="margin-bottom: 2em;">
+    <li>Explorative data analysis</li>
+    <li>Data visualization</li>
+    <li>Predictive models (classification and regression)</li>
+    <li>Clustering and segmentation</li>
+    <li>Data projection</li>
+</ul>
+
+<div class="panel panel-warning">
+    <div class="panel-heading">
+    <b>Students from our past courses said</b>
+    </div>
+    <div class="panel-body">
+        <div class="wrapper" style="padding-top:1em">
+            <div class="one">
+                <img style="float:left;" src="{% static 'courses/savannah_bw2.png' %}" alt="Savannah" />
+            </div>
+            <div class="two">
+                <blockquote style="overflow:hidden;">
+                    <p>"Introduction to Data Mining is an excellent course that uses Orange to teach students of all levels the fundamentals of data mining and machine learning. Through the fantastic visualization tools for viewing data and models, Orange made complex mathematical ideas much more intuitive and accessible. This course has allowed me to approach my research (neuroimaging analysis of psychiatric patients) in an entirely novel way! I would definitely recommend this course to anyone working with large data sets."
+                    </p>
+                    <footer>Savannah Gosnell</footer>
+                </blockquote>
+            </div>
+        </div>
+    </div>
+</div>
```

Comment:
I suggest some extra space after panel-warning. Title is too close to panel for my opinion.